src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2529 @@
+    - `old` (2.x default): as an array of tuple fields
+]])
+
+I['compat.c_func_iproto_multireturn'] = format_text([[
+    Controls wrapping of multiple results of a stored C function when returning
+    them via iproto:
+
+    - `new` (3.x default): return without wrapping (consistently with a local
+      call via `box.func`)
+    - `old` (2.x default): wrap results into a MessagePack array
+]])
+
+I['compat.console_session_scope_vars'] = format_text([[
+    Whether a console session has its own variable scope:
+
+    - `new` (4.x default): all non-local variable assignments
+      from the console are written to globals.
+    - `old` (3.x default): non-local variable assignments
+      are written to a variable scope attached to the console session.
+]])
+
+I['compat.fiber_channel_close_mode'] = format_text([[
+    Define the behavior of fiber channels after closing:
+
+    - `new` (3.x default): mark the channel read-only
+    - `old` (2.x default): destroy the channel object
+]])
+
+I['compat.fiber_slice_default'] = format_text([[
+    Define the maximum fiber execution time without a yield:
+
+    - `new` (3.x default): `{warn = 0.5, err = 1.0}`
+    - `old` (2.x default): infinity (no warnings or errors raised)
+]])
+
+I['compat.json_escape_forward_slash'] = format_text([[
+    Whether to escape the forward slash symbol "/" using a backslash
+    in a `json.encode()` result:
+
+    - `new` (3.x default): do not escape the forward slash
+    - `old` (2.x default): escape the forward slash
+]])
+
+I['compat.replication_synchro_timeout'] = format_text([[
+    The `compat.replication_synchro_timeout` option controls transaction
+    rollback due to `replication.synchro_timeout`.
+
+    - `new` (4.x default): A synchronous transaction can remain in the synchro
+      queue indefinitely until it reaches a quorum of confirmations.
+      `replication.synchro_timeout` is used only to wait confirmation
+      in promote/demote and gc-checkpointing. If some transaction in limbo
+      did not have time to commit within `replication_synchro_timeout`,
+      the corresponding operation: promote/demote or gc-checkpointing
+      can be aborted automatically
+    - `old` (3.x default): unconfirmed synchronous transactions are rolled back
+      after a `replication.synchro_timeout`
+]])
+
+I['compat.sql_priv'] = format_text([[
+    Whether to enable access checks for SQL requests over iproto:
+
+    - `new` (3.x default): check the user's access permissions
+    - `old` (2.x default): allow any user to execute SQL over iproto
+]])
+
+I['compat.sql_seq_scan_default'] = format_text([[
+    Controls the default value of the `sql_seq_scan` session setting:
+
+    - `new` (3.x default): false
+    - `old` (2.x default): true
+]])
+
+I['compat.wal_cleanup_delay_deprecation'] = format_text([[
+    Whether to use the option 'wal_cleanup_delay':
+
+    - `new` (4.x default): raise an error
+    - `new` (3.x default): log a deprecation warning
+]])
```

Comment:
Two `new` bullets.

Reply:
oops... The second should be `old`) Fixed.